internal/cmd/local/command_kafka_start.go

```diff
@@ -158,20 +154,25 @@ func (c *Command) prepareAndSaveLocalPorts(cmd *cobra.Command, isTest bool) erro
 			ControllerPort: "2999",
 		}
 	} else {
-		freePorts, err := freeport.GetFreePorts(4)
+		freePorts, err := freeport.GetFreePorts(3)
 		if err != nil {
 			return err
 		}
 
 		c.Config.LocalPorts = &v1.LocalPorts{
-			KafkaRestPort:  strconv.Itoa(freePorts[0]),
-			PlaintextPort:  strconv.Itoa(freePorts[1]),
-			BrokerPort:     strconv.Itoa(freePorts[2]),
-			ControllerPort: strconv.Itoa(freePorts[3]),
+			PlaintextPort:  strconv.Itoa(freePorts[0]),
+			BrokerPort:     strconv.Itoa(freePorts[1]),
+			ControllerPort: strconv.Itoa(freePorts[2]),
 		}
 
 		if kafkaRestPort, err := cmd.Flags().GetString("kafka-rest-port"); err == nil && kafkaRestPort != "" {
```

Comment:
Since `"8082"` is now the default value for `--kafka-rest-port`, it seems like users need to actively set `--kafka-rest-port ""` to reach the free port branch of this if block.

Do we want to always default to 8082? If so, maybe the else block can just be setting it to 8082 if the user overwrote it with the empty string.

Reply:
> Do we want to always default to 8082?

I think this makes the most sense, all of our other tools do this so this is what users expect.

Reply:
make sense. or i can just remove the else block, expecting user will not set this flag to "".

Reply:
the else block will act as fault tolerance if user unexpectedly input "" for tha flag

Reply:
Yup, but it just seems weird to me that it defaults to a free port and not just 8082 if they entered "".

Reply:
so do we want it to be still 8082 even if user inputs "", and only a free port when 8082 is not available?

Reply:
That seems reasonable to me.